daisy_latest_cloudbuild.yaml

```diff
@@ -1,4 +1,4 @@
-timeout: 600s
+timeout: 700s
```

Comment:
I would go with at least 900s (15min) to have some buffer

Reply:
Done